src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/ExclusionZoneIntegrationTest.kt

```diff
@@ -27,6 +30,7 @@ class ExclusionZoneIntegrationTest : IntegrationTestBase() {
   @BeforeEach
   fun setupClient() {
     webTestClient = webTestClient.mutate().responseTimeout(Duration.ofSeconds(60)).build()
+    govUkApiMockServer.stubGetBankHolidaysForEnglandAndWales()
```

Comment:
is this needed?

Reply:
Yes, we need it.
Because we are using ".uri("/licence/id/2")" this api which needs bankHoliday Service to return list of bank holidays'